### ISSUE TITLE
[Documentation] Keyboard commands

### ISSUE THREAD

**Issue by demilich1** [nushell/nushell#994]
I would like to have a list of all keyboard commands nushell supports. This may seem like a trivial issue, but it could be useful especially for more inexperienced users. While many commands are pretty much universal across all shells, like Tab for completion or using the arrows to navigate the command history, I think there is still value in documenting that.

Also in my concrete case, I noticed that when I start to type a command there is a suggestion in a different text color. My previous shell did not have that feature and it took me a while to find out that I could press 'arrow right' to actually use that suggestion.

**Comment by notryanb** [comment]
@demilich1 I agree. I'm wondering where the best place to document this would be?
I've started working on the [nushell cookbook](https://github.com/nushell/cookbook)
which is very minimal at the moment.

Another possibility would be the [docs section of the nushell site](http://www.nushell.sh/documentation.html). 

It might also be cool to have a `help` sub-menu which details all the shortcuts as well?

**Comment by demilich1** [comment]
I don't have a strong opinion whether the shortcuts should be in the cookbook or the docs section. However I agree it would be cool to have that information available inside the help menu as well!

**Comment by pwnorbitals** [comment]
The help menu sounds like a good place :)

**Comment by codyrobbins** [comment]
For the record, I was also highly confused by why it kept showing me previous commands while typing, seemingly as autocomplete suggestions, but neither `Tab` nor `Enter` nor anything else I tried would accept the completion. I would never in a million years have guessed it was right arrow. I then spent a bunch of time going through every chapter of the docs on the site but nothing talks about keyboard interaction; a bunch of googling was also fruitless until I finally stumbled on this issue.
